packages/graphql/lib/schema-builder/services/type-mapper.service.ts

```diff
@@ -67,18 +67,7 @@ export class TypeMapperSevice {
       );
     }
 
-    let isNotNullable: boolean;
-    if (isInputTypeCtx) {
```

Comment:
Since we no longer use the "isInputTypeCtx" argument, let's just remove it from the method signature.

Reply:
Good point. Have removed.